ci/install-kind.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -ex
+
+mkdir -p bin
+
+# nsenter is included on xenial
+
+# install socat (required by helm)
+sudo apt-get update && sudo apt-get install -y socat
```

Comment:
It is? But this was not added to the minikube script? I tracked down this to debug why it took so much longer to run the jobs with kind compared to using minikube.